README.md

```diff
@@ -24,20 +24,20 @@ Make sure you follow the instructions carefully. You will be asked to complete a
 ## 💭 Reflection and Self Assessment
 
 **I learned:** (repeat as needed)
-- .
+- .I learned about adding commits and pushing the assignment. Through the commits, it allows me to enter my comments about my actions onto my local machine, and the push will send my assignment to the cloud repository. 
 
 **I struggled with:** (repeat as needed)
-- .
+- .I struggled with the original set up. When I was trying to do the commands in the terminal, I kept getting the error that Git was not recognized. However, I didn't have Git installed and I wasn't logged into the GitHub app. Once I completed those two steps, I was able to add the commits and push the assignment. 
```

Comment:
This is my first time using Git as well, and I feel like I have a big learning curve when it comes to using it. You seem to have quickly figured out the commits and push concepts.